manifoldco/ui
# Add owner id to resource container

![image](https://user-images.githubusercontent.com/374078/75705623-5849f580-5c81-11ea-8d29-475167de39b8.png)

## Reason for change

We need to be able to display resource details when viewing the resource in a team context.




## Testing



The below requires changes to the Render dashboard in order to pass through the owner ID. Render also makes a query that needs to pass this along, so that also needs to be adjusted.

1. Make sure you're part of a team
1. Log into Render
1. Switch to your team context
1. Go to the Addons tab
1. If any resources exist, click on them
1. If a resource doesn't exist, create one, and then click on it in the Addons page
1. The resource should display successfully without errors

## Checklist




[docs]: https://ui.sandbox.manifold.co
[platform-testing]: https://app.gitbook.com/@manifold/s/engineering/playbooks/testing-your-code-against-a-platform
